src/main/java/com/spotinst/sdkjava/model/api/azure/statefulNode/ApiLaunchSpecImageConfiguration.java

```diff
@@ -0,0 +1,46 @@
+package com.spotinst.sdkjava.model.api.azure.statefulNode;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiLaunchSpecImageConfiguration implements IPartialUpdateEntity {
```

Comment:
ImageConfiguration - can be either marketplace or custom or gallery, so having private member - ApiLaunchSpecMarketplaceConfiguration is not correct.

Reply:
Added code for Custom and Gallery also.